deployment/k8s/README.md

```diff
@@ -74,11 +74,16 @@ By default, all of these components will be deployed, but if a user is certain t
 helm dependency update 
 ```
 
-The full Migration Assistant environment with Mock Customer Clusters can be deployed with the following helm commands
+The full Migration Assistant environment with source and target test clusters can be deployed with the following helm commands
 ```shell
 helm install ma -n ma charts/aggregates/migrationAssistant --create-namespace
-helm install mcc -n ma charts/aggregates/mockCustomerClusters
+helm install tc -n ma charts/aggregates/testClusters
+```
 
+Alternatively, specific test cluster configurations can be deployed instead of the default source and target configuration. Existing alternative configurations can be found in the `charts/aggregates/testClusters/environments` directory and deployed with helm commands similar to below
+```shell
+helm install tc-source -n ma charts/aggregates/testClusters -f charts/aggregates/testClusters/environments/es-5-6-single-node-cluster.yaml
+helm install tc-target -n ma charts/aggregates/testClusters -f charts/aggregates/testClusters/environments/os-2-latest-single-node-cluster.yaml
```

Comment:
This is cool! Is there a way to specify a alternative source and target? (can you just chain the `-f YAML_PATH` syntax?)

Reply:
Yep, the first half is what I was wondering about but what are/why do you need: `--set conditionalPackageInstalls.source=true --set conditionalPackageInstalls.target=true`?